src/ResourceManager/Profile/Commands.Profile.Test/TelemetryTests.cs

```diff
@@ -38,5 +42,25 @@ public void HashOfValidValueSucceeds()
             Assert.True(hash.Length > 0);
             Assert.NotEqual<string>(inputValue, hash, StringComparer.OrdinalIgnoreCase);
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void NetworkInterfaceWithEmptyAddressReturnsNull()
+        {
+            var networkInterfaceMock = new Mock<NetworkInterface>();
+            networkInterfaceMock.Setup(t => t.GetPhysicalAddress())
+                .Returns(
+                    () =>
+                        {
+                            return new PhysicalAddress(new byte[] { });
+                        }
+                );
+
+            var address = networkInterfaceMock.Object.GetPhysicalAddress();
+            Assert.NotNull(address);
+            Assert.Equal(string.Empty, address.ToString());
+            var hashAddress = MetricHelper.GenerateSha256HashString(address.ToString());
```

Comment:
I think the last two lines are the only lines needed in this test.  Mocking the network interface is only menaningful if we canuse it in the code.